src/components/AppMain.vue

```diff
@@ -48,7 +48,7 @@ export default {
     watch: {
         /* This is left here as an example in case the routes need to be debugged again
         '$route' (to, from) {
-            console.log(this.$window.isSameBaseRoute(from.fullPath, to.fullPath))
+            this.$log.debug(this.$window.isSameBaseRoute(from.fullPath, to.fullPath))
         },
         */
```

Comment:
These lines are commented out. After it is redirected to a debug-level logger, we could remove the comment and just let the logger ignore the logged data in the normal (non-debug) case. Am I right?

Reply:
Sure. Sounds like a plan.